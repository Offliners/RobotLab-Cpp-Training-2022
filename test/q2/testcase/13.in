888
BrOCHURE
aBOriGine
rElatively
SCaNDAL
FAme
pErfUme
Oak
cAUtIOUs
trANSIT
coNclUde
virtuous
4u070216
HERA5D
ALcOHOL
INCONVeNiENT
lEave
gREED
inTRiCaTE
RUG
among
d2nger
Chip
TRY
DOG
fifty
ESSENTIaL
RING
rISe
HelPFUl
paRtIal
Chat
motIvation
tutor
overdo
tiLL
PETITiOn
GOVERN
observatory
cHIMNEY
mAGNIFICENt
hUNGER
engIne
DIVERSIFY
MAId
KiTTEn
graveyard
lo5m
LIMITAtION
ORIGIN
S1ipment
DE84R52r7
tALE
fIvE
DISRegArD
ambIguOus
DISTRUST
cOlLeAGue
AWAIT
MANIFESTO
bAsED
stride
junE
fArm
POSTPONE
SURVEY
ANSWER
2r6078r
BREATHE
SIze
bIoLOGiCal
looSeLy
AbOrTION
8OMET
CoLor-Blind
pITCh
Era
HaLF
wEll
miniStry
martial
law
ROof
42S89790
lace
convene
CleANER
InsAne
VALuabLe
09589o69A4234
ALready
coming
aTTAINmeNT
REUnion
dilute
REGULATE
pleasant
cUStoMARy
E9Rl4
convert
R05e
CompreSs
SuBtrACT
CHEF
film
humid
sister
ViSUal
l
gliMPse
Silly
beloNg
SHarE
toWn
PENCiL
ApPOinT
CHIldbirTH
iN902M6L
cONTEnT
CABiN
limb
brItON
cOMpatIbLy
tR3spAss
vIruS
allergy
ADopT
femaLE
SAdDLe
sALaRY
noodle
7I4ve5
DouGhnut
unrest
discuss
BLanKeT
P04Ts
cHoLeSterOl
EXpLOrATIOn
sUNny
MAnneR
SUmMER
cheap
civiLIAN
cATErPILlAR
DiffEreNcE
FLATTER
AdApt
stone
implement
Gas
INTENT
PeRCENT
CEREMonY
qualiTy
vaLIDITY
flueNT
64042g1t84
pItY
crane
M7sUNdER208n1I99
c69
Duck
pOtTeR
giV9
SuRrOUnD
bottle
begGaR
2215p91
ImpOSING
AIrmaIL
BESIDES
4046827io0
ShaKE
PoSTUrE
admiRal
uNCLe
goldEN
plAster
sPe6K
PeAk
fLEE
d03Ect172
HYPHEN
appRopriate
enSUe
murmur
mill
boy
prOcess
grovE
TORTURE
toleraTE
TASTY
tentative
signiFicant
7802r7252
ANCIenT
LOTiON
PEsSIMISM
bridge
UNPRECEDENTEd
COMMERCiAl
CRAWl
FUnD
rubber
LikeWIse
SOcceR
tRIcKY
mother
DiStRib2T6o1
TECHNOLoGY
de8m
justly
LonEly
sPleNdId
cEleBrAte
PRESENTAtIoN
SeRuM
CA6VE
dE217TIoN
rePEtITION
BRIDEgROOM
bill
BIND
surprisInG
PoND
ANXIETY
bOmb
joIn
FAMILIAr
sYsTem
comet
sLIP
EXPLORE
haRDware
antHEm
disCourage
historic
request
mARRIEd
disSIDeNT
SyMMeTRY
auSTRaLiA
CLIFF
DREaD
sUpPORT
CONSPIRACY
CUSHIOn
circLe
prOFIT
ruLer
INJUSTICE
wALl
CASINO
siMIlARLy
fUlfill
prEDiCTIoN
heaRt
1I2a5ree
PIOUS
ExTRemE
SneaK
bEHIND
siMilarity
SPICE
prevEntiVe
fUnny
9067I22
welL
RaW
owNErShiP
bet
PLOw
0325N
tiMbEr
MEAN
INVeNTIoN
KINg-SIZe
PENETRATING
overTake
TRAIN
coUrTeous
N
BEING
E
ThErAPEUTiC
rePlACeAbLE
AIdE
N4waDAy3
up
dEGREe
livIng
line
aLCoHOLIc
sore
fantastic
brain
mUsIcAL
BLAmE
crude
ACcuSATIOn
enOugh
WEALTHY
CHILDHOOD
lEMOnAde
exclaim
BIkini
PROpHeSY
ReAsonabLe
neighborHOod
DOLPhIn
LAboRATOrY/LAb
sAFety
peril
CaUSE
cROSSING
CAtER
endurance
MANiPULATe
actor
featURE
GOddEss
RObE
cOmmittee
BuG
TECHNIQuE
allEGE
LeVel
mAndaRiN
GUN
SILvER
Venus
BaskeT
BufFet
mU244r
thIn
Build
changeabLe
ARMchaiR
CONGRATUlATE
V2c63ION
AverAGE
libRAry
examine
sPEciAl
FabLE
al1NG
autObiogRapHy
GOLD
noMINaTe
UNFOrGETTABLE
7e2n266
DISSUADE
INtEnTIOn
f3Ce
expertise
SMOKE
FORgeTful
warn
ReAlitY
BOdYGUARd
t90a9
B68lY
veRSE
LAUNDRY
assertive
inefFECtIvE
Canal
cHeesE
RID
reGULar
shrink
PURELY
sHAPE
ESSEnce
WHATevEr
579829e
briDe
PropErty
aesthetIc
ACcUMuLaTE
078NO6s
LesSER
DEVOTiON
CHoiCE
DelicaCY
NAturE
dispersE
INVITATION
SpECIEs
ApROn
PeRFoRmANCE
CorRUPTIoN
tOMaTo
cOntRibutiOn
sEvErEly
Odds
ADVisOR3
PLAiNly
CONtINUatIOn
o
Philosophy
StYLE
OUSt
MiLliGrAm
HiP
CONClUSION
RefusE
cOnSulT
allow
6285T6
SHOrts
5742a0
MUCh
sEmInAR
alaS
PACT
11aIlabi9iT0
ScRAP
OrIeNTAL
THUS
RemArK
underestimAte
MASK
P5RC4
refund
taIl
eFFORT
tr5vi8L
tarGeT
90566B78
whiSper
IntensivE
straightEn
AppRAiSAL
postman
DenoUNce
SCARLET
TooTh
OCCASIONAL
DOcumENT
AnGler
iRrEgUlar
along
WHICHEVER
prOPosaL
DedUCTion
MyTHOLoGY
APPlICABLe
CONSECutiVE
MembeRsHIP
ESSAY
bicycle/biKe
pionEeR
thIrd
vanish
stepchild
COnvIcT
console
LiZARD
masTEr
lIE
ho0Y
2808a0
Squad
soDium
InjuRE
eMBaRRass
CHARMING
nakED
rEst0r4Tion
dIAPEr
lEan
coLleGE
oV671aKE
NAT0R7LLY
SCULPTURE
walnut
eMIgRaNT
f41l
suck
COUPLE
cRackdown
afForD
1on2995e
FASHION
hUNGEr
sliDe
beam
u
maNifestAtion
OVERLoOK
BUCK/DOLLAR
rARe
prOsper
eeL
S8W
cOmplete
fActORy
ariSe
coke
PURcHASE
TeLegRam
SCALe
VEX
ThiRST
QuAInT
pA151E
waste
numerous
PARAGRAPH
papA
hAIR
breakdOwn
syllabUS
hYphen
calcUlatoR
LABOr
HoNESTy
B40757
TOwel
tray
natIoNality
FAVOriTE
depEnDeNt
roUGHly
pain
such
ViOLaTE
HISTORiAN
al3S
PROnE
DIVIDEND
liABIliTy
pollutANt
WAnt
NecK
how
TaBlespoOn
SeVEral
36ssio2363
MODErNIZATION
TiCKET
knob
ECLIPSE
Harness
rETIrEMENT)
rainfall
cUrve
EXTENSIVE
ASPIRING
MORTAR
HOMESICK
O
hungrY
DiGESt
SAND
anY
PROSE
TRIAL
NOTE
CHARITY
NoBlE
DISCLOSE
tHROUghouT
DART
EXCITE
EAgER
carROt
gAze
howeVer
448Nt3669T239
breAst
blaz2
accord
TENSe
textBook
REVErSE
HOt
glove
GoldfIsH
QuarTer
BROWN
linGer
CoUNTAblE
AscrIbe
SEnD
manpower
PrELimINARY
4E710
relinQUish
CONNeCTION
sIMulTaneOUsly
stoRAGE
towEr
c14416467
illustRAte
Boost
inDIVIDuAL
i825R6
ComMuniST
nomination
EXCeL
BuNKER
STeaM
ClARifY
diSpLAY
usual
chaiR
pRiVa11
RELAX
Bird
NOISE
POLITICIAN
ThOUgH
SpONSOR
aLmOsT
007i15eR3
vOLCAno
COnsoLATiON
372ID1
950Er63
ANgULar
obSerVATION
AuTONoMY
MotHERHOOD
veRY
PReSumptiOn
gasoline
HouSewoRk
HYPOtHESIS
ReCEPtIOn
STUPID
obseRve
coURt
ambulance
invite
cOLLAPSE
TOU4H
hAnDY
SOvEREIGn
DRAFT
CO660521534
enlarGe
UnFOrtUNATE
cAPE
FORTUNATE
DOSE
dEter
EARNEST
juDiCial
congressional
frienDship
DISQuaLIFY
GraceFUl
SPIrItUAl
oBSesSion
tRaNSlATE
inFEr
aCcompany
T
deciSive
INfLUenzA
SauCe
RELATiVE
cROp
HAIrdo
naRrow
coeDucAtIoN
slum
lamb
pOsITiVE
4atRIOt
anonymOus
AnecDoTE
eMERGENCY
Unlo71
BETTEr
NuRSERY
curry
1An64s
S142ION101
competitioN
SWAlLow
accLAim
RuN
mORNIng
senSAtiOn
moDeRnize
newSpapEr
DEARLY
tHunDerstorm
TROUsERs
4ODdEsS
dEPrEsSion
InHeRIT
frog
ceLEbRATIOn
victory
63e63
diG
CrUSH
SWEATeR
3900a5
ROT
PoRtion
bunk
bEgiNNER
MoraL
N
poLYnoMiAL
PoLE
PremISe
drama
POSSESS
gear
ROTTEN
STEReo
kInD
spoT.
CLoCK
2o4k
hUrt
sTrAWbErRy
o
ABUNDANCE
93719r83n7
QUOTE
REHABiLItATiOn
SOMeThiNG
DENY
sIgnature
AftErNooN
vIA
RElATIOnSHiP
BRiGhTeN
eXpeRt
invisible
Meadow
soRT
HEaD-On
TReAT
oVERnIGhT
PLEASED
SHAMPOo
ACCESSIBLe
Far
FLoaT
ExCeEdINgLY
PrINTER
fragmEnt
cOnstructioN
PENGUin
OBJEcTIoN
NOURIsh
COnSTITUTIOn
EVENING
cALEndAr
S06ua37
CROW
greeT
plate
L
blue
STAND
GRIM
TAnGERIne
ReSpoNsIBilitY
LiLY
blacK
SHaDOW
HAIl
OTheRWiSe
defEcT
bla0T
a
prOsPect
lIpSTiCk
FUnCTIONal
rUggEd
bEgin
TonGue
r
DAmAgE
NowAdAys
heightEN
surpRised
snowy
Camel
TELEGRAPH
spectacular
roma2ce
opPRESS
SAiL
trust
aCRoSS
grAiN
DARLING
pLAYGrOUnD
Be821f
925305R96369
sUDdeN
miLItARY
nEcesSITY
dIsCOMfORT
CeRtain
seLecTion
lAND
circulaTiOn
NUCLeUs
celL
bRAVEry
loan
ChaNCELLoR
StiMulUs
seRMoN
BEG
syStematic
heTEROSexuAl
mAScULiNE
engiNeERInG
TidE
FORMATION
LuNg
INNOCENT
sUperfIcIAL
34EcKB03k
BAcKGround
s
KNife
belIeF
HaLl
arch
defeNse
conSeNsuS
lawyer
BURGLAR
egg
biblicaL
PuNCTUAL
hUnTeR
TALL
AtTIC
lAsEr
stack
fifth
skin
SUit
Stroke
GIRL
reservoir
8URL
INTERRUPT
8Ea0
GRAPE
9ire
ascertAin
HEEL
oVerlAP
salad
HAPPINESS
teN
compaRATivElY
BAIT
prOfEsSIONal
shIRt
sUPPOSEDLY
VEHICLE
bury
bItter
tr390e7
cAPACITY
REcruiT
FACuLTY
CuBIC
blossom
COlLABORATE
INcOMplete
so
copilot
WeARy
hold
5unger
5547800/AD01S19
HEIGHT
seCtIoN
saw
ba6K00PEr
