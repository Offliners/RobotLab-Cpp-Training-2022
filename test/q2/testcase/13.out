n. 小冊子
n.  原著居民
adv. 相對地
n. 醜聞
n./v. 聲譽
n. 香味；香水
n. 橡樹
adj. 小心的
n. 運輸
v. 結束；斷定
adj. 品德高尚的
Unknown
Unknown
n. 酒精
adj. 使人不便的
n./v. 休假；許可；離開
n. 貪婪
adj. 複雜的
n. 小地毯
prep. 在…之中
Unknown
n./v. 晶片；削
n./v. 嘗試
n. 狗
adj./n. 五十
adj. 必要的；精華的
n./v. 圈；圍住
n./v. 上升；站起來
adj. 有幫助的
adj. 局部的
v./n. 聊天
n. 動機
n. 家庭教師
v. 做太多或過份
prep./conj. 直到
n. 請願(書)
v. 統治；治理
n. 天文臺；氣象臺
n. 煙囪
adj. 壯麗的
n. 飢餓
n. 發動機；引擎
v. 使多樣化
n. 少女
n. 小貓
n. 墓地
Unknown
n. 限制
n. 起源；由來
Unknown
Unknown
n. 故事
adj/n. 五
v. 漠視
adj. 含糊的
v. 不信任
n. 同事
v. 等候；期待
n. 宣言；聲明
adj. 以…為基礎的
v. 邁大步走
n. 六月
n. 農莊；農場
v. 延期
n./v. 調查；審視
n./v. 答案
Unknown
v. 呼吸
n. 尺寸
adj. 生物(學)的
adv. 不嚴格地；寬鬆地
n. 墮胎
Unknown
adj. 色盲的
n. 瀝青
n. 年代
adj./adv./n. 二分之一的
adv. 很好地
n. 牧師之職；內閣
adj. 軍事的
n. 法律
n. 屋頂
Unknown
n. 鞋帶；花邊
v. 召集
n. 清潔工(劑)
adj. 瘋狂的
adj. 珍貴的
Unknown
adv. 已經；先前
n. 到來
n. 達到
n. 重新結合；聚會
v./adj. 稀釋(的)
v. 限制；控管
adj. 令人愉快的
adj. 習慣上的
Unknown
v. 轉換
Unknown
v. 壓縮
v. 減掉
n. 主廚
n./v. 軟片；電影
adj. 潮溼的
n. 姊妹
adj. 視覺的
Unknown
v./n. 看一眼；瞥見
adj. 愚蠢的
v. 歸屬於
n./v. 分擔；分配
n. 城鎮
n. 鉛筆
v. 任命；指派
n. 分娩
Unknown
adj./v. 滿足的
n. 客艙
n. 肢；臂
n. 英國人
adv. 和諧地
Unknown
n. 病毒
n. 過敏性反應
v. 採取；收養
adj./n. 女的；雌性的
n. 馬鞍
n. 薪水
n. 麵條
Unknown
n. 甜甜圈
n. 動亂
v. 討論
n. 毛毯
Unknown
n. 膽固醇
n. 研究
adj. 陽光充足的
n. 方法；態度
n. 夏天
adj. 便宜的
n. 平民；文職人員
n. 毛蟲
n. 差異
v. 諂媚；奉承
v. 使適應
n./v. 石頭
v. 實現
n./v. 氣體；瓦斯
n. 目的
n. 百分之…
n. 典禮；禮儀
n. 質量
n. 有效力
adj. 流暢的
Unknown
n. 可惜
n. 起重機；鶴
Unknown
Unknown
n. 鴨子
n./v. 陶藝家；閒逛
Unknown
v. 圍繞
n./v. 瓶子；把…裝瓶
n. 乞丐
Unknown
adj. 氣勢雄偉的
n. 航空郵件
prep./adv. 除此之外
Unknown
n./v. 搖動
n. 姿勢
n. 海軍將官
n. 伯父
adj. 金色的；貴重的
n. 灰泥
Unknown
n. 山頂
v. 逃走；逃離
Unknown
n. 連字號
adj. 適當的
v. 接著發生
n./V. 私語
n./v. 磨坊；碾碎
n. 男孩；少年
n./v. 過程；加工
n. 樹叢
v. 拷打；折磨
v. 容忍
adj. 美味的
adj. 試驗性的
adj. 重要的
Unknown
adj. 古代的
n. 外用藥水；化妝水
n. 悲觀(主義)
n./v. 橋樑；艦橋
adj. 前所未有的
adj./n. 商業的
v. 爬行；蠕動
n./v. 資金
n. 橡膠
adv. 同樣的
n. 足球
adj. 狡猾的
n. 母親；媽媽
Unknown
n. 工業技術；(一門)科技
Unknown
adv. 合理的
adj. 寂寞的
adj. 輝煌的
v. 慶祝
n. 表演
n. 血清
Unknown
Unknown
n. 重覆
n. 新郎
n. 帳單；鈔票
v. 捆；綁
adj. 使人驚訝的
n. 池塘
n. 焦慮
n./v. 炸彈；轟炸
v. 參加
adj. 熟悉的
n. 系統
n. 彗星
n./v. 滑倒
v. 探索
n. 五金器具；硬體
n. 聖歌
v. (n)使洩氣；使沮喪
adj. 歷史上著名的
n./v. 要求
adj. 已婚的
adj./n. 持異議的(人)
n. 對稱(性)
prop./n. 澳洲
n. 懸崖
n./v. 害怕
n./v. 支持
n. 陰謀
n. 墊子
n./v. 循環；畫圓圈
n./v. 利益；有益於
n. 統治者
n. 不公正
n. 牆
n. 賭場
adv. 同樣地
v. (n)實現
n. 預言
n. 心臟；中心
Unknown
adj. 虔誠的
adj./n. 盡頭的；極限
v. 鬼鬼祟祟地走
prep./adv. 在…後面；落後
n. 相似之處
n./v. 香料；加香料於
adj. 預防的
adj. 有趣的
Unknown
adv. 很好地
adj. 生的
n. 所有權
n./v. 打賭
v. 耕(地)
Unknown
n. 木材
adj. 吝嗇的
n. 發明；創造力
adj. 特大號的
adj. 敏銳的
v. 追上
n. 火車
adj. 有禮貌的
Unknown
n. 存在；生物
Unknown
adj. 有療效的
adj. 可取代的
n. 助理
Unknown
prep./adv. 向上地
n. 度數；學位；等級
adj. 活著的
n./v. 線條；畫線於
n./adj. 酒精飲料；含酒精的
adj. 疼痛的
adj. 想像中的
n. 腦袋；智力
adj./n. 音樂的
n./v. 責備；歸咎於…
adj. 未加工製造的
n. 控訴
adj./adv./n. 足夠的
adj. 富裕的
n. 童年時期
n. 檸檬水
v. 驚叫
n. 三點式泳裝
v. 預言
adj. 有理由的；合理的
n. 鄰近地區
n. 海豚
n. 實驗室
n. 安全性
n. 危險
n./v. 導致；原因
n. 交叉口
v. 承包伙食
n. 耐力
v. 操縱
n. 男演員
n./v. 特徵
n. 女神；受尊崇的女子
n. 長袍
n. 委員會
n. 蟲子
n. 技術
v. 聲稱
n. 高度；層；水準
n. 國語
n. 槍；砲
adj./n. 銀製的；銀
n. 維納斯女神；金星
n. 籃子；簍子
n. 自助餐；快餐
Unknown
adj. 薄的
v. 建造
adj. 易變的
n. 扶手椅
v. 恭喜
Unknown
adj./n./v. 平均的
n. 圖書館
v. 檢查；調查
adj. 特別的
n. 寓言；虛構的故事
Unknown
n. 自傳
n./adj. 黃金(色)
v. 提名某人；指派
adj. 難忘的
Unknown
v. 勸阻
n. 意圖；目的
Unknown
n. 專業知識
n./v. 煙；抽煙
adj. 健忘的
v. 警告
n. 真實性
n. 侍衛
Unknown
Unknown
n. 詩節
n. 洗衣店
adj. 武斷的
adj. 無效力的
n. 運河
n. 乳酪；乾酪
n./v. 解除；免談
adj. 整齊的；規律的
v. 縮小
adv. 完全地
n./v. 形狀；做成
n. 本質；精華
adj./advJpron./conj.  任何的；無論什麼
Unknown
n. 新娘子
n. 財產
adj. 美感的
v.  累積
Unknown
adj./adv. 較小的
n. 奉獻
n./adj. 選擇
n. 精緻
n. 自然
v. 使散開
n. 邀請
n. 種類
n. 圍裙；工作裙
n. 成就；表演
n. 道德敗壞
n. 蕃茄
n. 貢獻；捐獻
adv. 嚴重地
n. 可能的機會
Unknown
adv. 清楚地
n. 繼績
Unknown
n. 哲學
n. 風格
v. 驅逐
n. 毫克
n. 臀部
n. 結論；推論
v. 拒絕
v. 與…商量
v.  允許
Unknown
n. [pl. ]短褲
Unknown
adj./adv./n. 許多的
n. 研討會
interj. 哎呀(表悲傷或恐懼)
n. 協定；公約
Unknown
n. 碎片
n./adj. 東方人；東方的
adv. 這樣
n./v. 論述；評論
v. 低估
n./v. 面具
Unknown
n. 退款
n. 尾巴
n. 努力
Unknown
n./v. 靶子；對·瞄準
Unknown
n./v. 低語
adj. 加強的
v. 清理
n. 評價
n. 郵差
v. 指責
adj. 鮮紅色的
n. 牙齒
adj. 偶爾的
n./v. 文件
n. 釣魚者
adj. 不合常規的
prep./adv. 沿著；一起
adj./pron. 無論哪個
n. 計劃；建議
n. 推斷
n. 神話學
adj. 適用的
adj. 連續的
n. 會員(資格)
n. 散文；論說文
n. 腳踏車
n./v. 拓荒者；開拓
adj./n. 第三
v. 消失
n. 妻與前夫(或夫與前妻)所生的孩子
v./n. 證明有罪；囚犯
v. 安慰
n. 蜥蜴
n. 主人；雇主
n./v. 躺臥
Unknown
Unknown
n. 爭吵
n. 鈉
v. 傷害
v. 使出糢
adj. 有魅力的
adj.  裸體的
Unknown
n. 菱形花紋(布)；尿布
v. 傾斜；倚靠
n. 大學；學院
Unknown
Unknown
n. 雕塑(品)
n. 胡桃(樹)
n. 移民
Unknown
v. 吸；吮
n. (一)對
n. 取締
v. 類得起；承擔得起
Unknown
n./v. 時尚；風氣
n. 飢餓
n./v. 打滑
n. 樑；光束
Unknown
n. 顯示；發表
v. 俯視
n. (美)元
adj. 珍貴的
v. 興旺
n. 鰻
Unknown
adj./v. 完全的；完成
n. 工廠
v. 升起；復活
n. 可口可樂
n./v. 購買
n. 電報
n. 標準
v. 使惱怒
n. 口渴
adj. 奇特的
Unknown
n./v. 浪費
adj. 許多的
n. 段落
n. 爸爸(非正式)
n. 毛髮；頭髮
n. 故障
n. 教學大綱
n. 連字號
n. 計算機
n./v. 勞動；勞工
n. 誠實
Unknown
n. 毛巾
n. 盤子
n. 國籍
adj./n. 特別喜愛的
adj. 依賴性強的
adv. 大體上
n. 疼痛
adj./prop. 如此的
v. 違背；侵犯
n. 歷史學家
Unknown
adj. 易於發生(不愉快事)的；有傾向的
n. 股息
n. 有義務
n. 污染物
n./v.  想要
n. 頸；脖子
adv. 如何；多少
n. 大湯匙
adj./n. 幾個的；幾個
Unknown
n. 現代化
n. 票
n. 節；瘤
n. (日、月)蝕
n. 馬具
v. (n)退休
n. 降雨
n. /v曲線；使彎曲
adj. 廣大的
adj. 有抱負的
n. 灰泥
adj. 懷鄉病的
Unknown
adj. 飢餓的；渴求的
v. 消化
n. 沙
adj./adv./pron. 任何的；若干
n. 散文
n. 審判
n./v. 筆記；注意
n. 慈善團體
adj. 高貴的；貴族的
v. 露出
prep./adv. 遍及
n. 鏢；急衝
v.(n. )使興奮
adj. 熱切的
n. 胡蘿蔔
v. 凝視；注視
adv./conj. 然而
Unknown
n. 胸部
Unknown
v. 符合
adj. 緊張的
n. 教科書
adj./n./v. 相反的；反轉
adj. 熱的；熱情的
n. 手套
n. 金魚
n. 四分之一
adj./n. 棕色的
v. 逗留
adj. 可數的
v. 把…歸因於
v. 送
n. 人力
adj./n. 初步的；初步行動
Unknown
v. 放棄
n. 連接；關聯
adv. 同時
n. 儲存
n./v. 塔樓；屹立
Unknown
v. 圖解；說明?
v. 提升
adj./n. 個別的；個人
Unknown
n./adj./n. 共產主義的；共產主義者
n. 提名；任命
v. 擅長
n. 媒倉；掩體
n./v. 蒸汽
v. 澄清
n./v. 陳列
adj. 通常的
n. 椅子
Unknown
v. 放鬆
n. 鳥
n. 噪音
n. 政治家
adv./conj. 然而；雖然
v. /n贊助(者)
adv. 幾乎
Unknown
n. 火山
n. 慰藉
Unknown
Unknown
adj. 有尖角的
n. 觀察
n. 自治(權)
n. 母性
adv. 非常
n. 推測
n. 汽油
n. 家事；家務
n. 假設
n. 接收
adj. 愚蠢的
v. 觀察
n. 法庭
n. 救護車
v. 邀請
v./n. 崩塌
Unknown
adj. 手邊的；巫巧的
n. 元首
n./v. 草稿
Unknown
v.(n. )擴大
adj. 不幸的
n. 披肩；海角
adj. 幸運的
n. (藥物等的)一劑
v. 制止
adj. 認真的
adj. 法庭的
adj. 國會的
n. 友誼
v. 取消資格
adj. 優美的
adj. 精神的；心靈的
n. 成見；頑念
v. 翻譯
v. 推斷
v. 陪同
Unknown
adj. 果斷的
n. 流行性感冒
n. 醬料
adj. 比較上的
n. 作物
n. 髮型
adj./v. 狹窄的；限制
n. 男女合校的教育
n. 貧民窟
n. 小羊
adj. 確信的；積極的
Unknown
adj. 不具名的
n. 軼事
n. 緊急情況
Unknown
adj./adv. 更好的
n. 育嬰室
n. 咖哩食品
Unknown
Unknown
n. 競賽；競爭
n. 燕子
v./n. 歡呼；歡呼聲
v. 跑
n. 早晨
n. 感覺
v. 使現代化
n. 報紙
adv. 深切地
n. 雷雨
n. 褲子
Unknown
n. 沮喪
v. 繼承
n. 青蛙
n. 慶祝
n. 勝利
Unknown
v.  挖；掘
v./n. 弄碎；壓碎
n. 毛線衫
Unknown
v./n. 腐敗；腐朽
n. 一部分
n. 臥鋪
n. 初學者
adj. 道德上的
Unknown
n. 多詞學名；多項式
n. 南極或北極
n. 前提
n. 戲劇
v. 擁有
n./v. 齒輪；搭上齒輪
adj. 腐敗的
n. 立體聲；立體音響器材
adj. 親切的
n./v. 地點；散布
n. 時鐘
Unknown
n./v. 受傷
n. 草莓
Unknown
n. 豐富
Unknown
v. 引用
n. 康復
pron. 某物
v. 否定；否認
n. 簽字
n. 下午
prep. 取道
n. (親戚)關係
v. 照亮
adj./n. 熟練的；專家
adj. 看不見的
n. 草地
n./v. 種類；把·分類
adj./adv. 正的
n./v. 請客；對待
ad v. 在昨夜
adj. 高興的
n. 洗髮(精)
adj. 易接近的
adj./adv. 遙遠的
v. 飄浮；搖擺
adv. 極其
n. 印刷業從業人員
n. 碎片；未完成的部分
n. 建造物；建設
n. 企鵝
n. 反對
v. (n〕滋養
n. 憲法；法規
n. 晚上
n. 日曆；行事曆
Unknown
n. 烏鴉；雞啼聲
v. 問候
n. 盤；碟
Unknown
adj./n. 藍色的
n./v. 攤位；站
adj. 嚴厲的
n. 橘子
n. 責任
n. 百合花
adj./n. 黑的
n. 陰影
v. 歡迎
adv. 除此之外
n. 缺點
Unknown
Unknown
n. 期望
n. 口紅
adj. 起作用的
adj. 崎嶇的
v. 開始
n. 舌(頭)
Unknown
n./v. 損害；毀壞
adj. 現在；今日
v. 增高
adj. 感到驚奇的
adj. 多雪的
n. 駱駝
n. 電報
adj. 壯觀的
Unknown
v. 壓迫；壓制
n./v. 船；坐船旅行
n./v. 信任
prep./adv. 橫越
n. 穀粒；穀物
adj./n. 親愛的
n. 操場
Unknown
Unknown
adj. 突然的
adj./n. 軍事的；軍隊
n. 必需；必需品
n. 不舒適
adj. 確信的
n. 選擇
n. 土地
n. 循環；流通；貨幣
n. 中心；核心
n. 小囚房；細胞
n. 勇敢；勇氣
n./v. 借出
n. 司法官
n. 刺激(物)
n. 說教；佈道
v. 乞求；乞討
adj. 有系統的
adj. 異性戀的
adj. 男性的；陽性的
n. 工程
n. 潮汐
n. 形成；組成
n. 肺
adj. 清白的；無知的
adj. 表面(皮)的
Unknown
n. 背景
Unknown
n. 刀
n. 信念；信條
n. 大廳；走廊
n. 拱門；拱形
n. 防衛
n. 多數人的意見
n. 律師
n. 竊賊
n. 蛋；卵細胞
adj. 聖經的
adj. 如期的；準時的
n. 獵人
adj. 高的
n. 閣樓
n. 雷射
v. 使成堆
adj./n. 第五(的)
n./v. 皮膚；剝…的皮
n. 一套衣服
n. 一擊
n. 少女
n. 蓄水池
Unknown
v. 打斷
Unknown
n. 葡萄
Unknown
v. 查明
n. 腳後跟
v. 部分重疊；部分相同
n. 沙拉
n. 快樂
n. 十
adv. 比較地
n./v. 誘餌；引誘
adj./n. 專業的
n. 襯衫
adv. 據推測
n. 車輛
v. 埋
adj. 有苦味的；痛苦的
Unknown
n. 容量；能力
v. 招募
n. 官能；技能
adj. 立方的
n. 花(叢)
v. 合作
adj. 不完全的
adv./conj. 所以
n. 飛機副駕駛
adj. 筋疲力竭的
v./n. 抓住；支撐
Unknown
Unknown
n. 高度
n. 地域
n. 鋸
Unknown
